front/src/components/Form.vue

```diff
@@ -181,13 +180,11 @@ export default {
       }
       this.getProjects = debounce(this.getProjects, 500)
     }
-    httpClient.get('/myTimesheetUrl')
+    httpClient.get('/redmineBaseUrl')
```

Comment:
Eh oui timesheetbaseurl est redevenu baseurl comme je l'avais pensé au départ au cas où on aurait eu d'autres usages...

Reply:
Yes but.. if you look at 36d5167e8f3f521513af3b4e67e8dff5c719c7e6 you'll see that the url was constructed on the **server side** and therefore was not a baseUrl :p

Reply:
Je l'ai appelé comme ça pour une bonne raison quand même, j'avais commencé à le construire de l'autre côté, d'où le nom de variable. Le tout en me disant qu'il serait tjrs temps de changer si j'en avais besoin pour autre chose.